pkg/trace/api/profiles.go

```diff
@@ -34,6 +34,13 @@ func profilingEndpoints(apiKey string) (urls []*url.URL, apiKeys []string, err e
 	main := profilingURLDefault
 	if v := config.Datadog.GetString("apm_config.profiling_dd_url"); v != "" {
 		main = v
+		if strings.HasSuffix(main, "/v1/input") {
+			// backwards compatibility with v1 intake
+			main = strings.Replace(main, "/v1/input", "/api/v2/profile", 1)
+		} else if !strings.HasSuffix(main, "/api/v2/profile") {
+			// handle user only specifying base intake url
+			main += "/api/v2/profile"
```

Comment:
would this be ok if the user gives an incorrect path (e.g. `https://intake.profile.datadoghq.com/api/v3/profile`) because then it would tack on to the end and be `https://intake.profile.datadoghq.com/api/v3/profile/api/v2/profile`. But we would return an error either way because neither resolve as an intake path...